### PR TITLE
Add cmdlet Get-AzServiceTag

### PR DESCRIPTION
**Pull request by anton-evseev**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by number213** [comment]
cc: @MikhailTryakhov @chandrasekarsrinivasan

File: src/Network/Network/Models/PSServiceTagInformation.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+namespace Microsoft.Azure.Commands.Network.Models
+{
+    public class PSServiceTagInformation
+    {
+        [Ps1Xml(Label = "Name", Target = ViewControl.Table, Position = 1)]
+        public string Name { get; set; }
+
+        public string Id { get; set; }
+
+        public string ChangeNumber { get; set; }
```
Comment (chandrasekarsrinivasan) [inline src/Network/Network/Models/PSServiceTagInformation.cs]: Can you add servicetagproperties here?

Comment (number213) [reply src/Network/Network/Models/PSServiceTagInformation.cs]: @chandrasekarsrinivasan I've added `x-ms-client-flatten` in Swagger to move props from `properties` up one level, how it's done for other resources with nested properties. Was that wrong?

Comment (chandrasekarsrinivasan) [reply src/Network/Network/Models/PSServiceTagInformation.cs]: Yeah, don't move it one level. This is not a top-level object like other properties in NRP.

**Comment by MikhailTryakhov** [comment]
@number213 why is it failed for so many jobs?

**Comment by number213** [comment]
Merged from https://github.com/Azure/azure-powershell/pull/9292 to resolve build errors due to InterfaceEndpoints being deprecated. Once https://github.com/Azure/azure-powershell/pull/9292 is merged this should be good to go

Cmdlet review: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/308

**Comment by number213** [comment]
@MiYanni updated PR, please take a look

**Pull request merged**